code/player/TTTPlayer.Shop.cs

```diff
@@ -0,0 +1,45 @@
+using Sandbox;
+
+using TTTReborn.Items;
+
+namespace TTTReborn.Player
+{
+    public enum BuyError
+    {
+        None,
+        InventoryBlocked,
+        NotEnoughCredits
+    }
```

Comment:
Neat I like this.

Reply:
For better clientside feedback in the shop later :D